data/bingo/functions/util/bingo_menu/print_options_hint.mcfunction

```diff
@@ -0,0 +1,9 @@
+#> bingo:util/bingo_menu/print_options_hint
+#
+# Display the bingo menu options hint to @s
+#
+# @internal
```

Comment:
`@within function bingo:util/bingo_menu/print_menu`? Or probably just inline it into that function? I don't think there really is a desire for calling this function without printing the menu before.

Perhaps also change `print_menu_hint` to call `print_menu` and rename `print_menu` -> `print_with_hint`, `print_options` -> `print_without_hint`, `print_menu_hint` -> `print_with_menu_hint`

Reply:
## Current
This is the current functionality of the files:

- `print_with_menu_hint`:
```
Click here to go back to the lobby.
Click here to switch to spectator mode.
Hint: You can click [≡] or use the command /trigger bingo.menu at any time to display the bingo menu.
```
- `print_without_hint`:
```
Click here to go back to the lobby.
Click here to switch to spectator mode.
```
- `print_with_hint`:
```
Click here to go back to the lobby.
Click here to switch to spectator mode.
Hint: You can also use the command /trigger bingo.lobby at any time manually to return to the lobby or /trigger bingo.spectator for spectator mode.
```

## Proposed
Always include the hint in parentheses, so that the other print statement is not needed?

- `print_with_menu_hint`:
```
Click here to go back to the lobby (Or run /trigger bingo.lobby)
Click here to switch to spectator mode (Or run /trigger bingo.spectator)
Hint: You can click [≡] or use the command /trigger bingo.menu at any time to display the bingo menu.
```
- `print_without_hint`:
```
Click here to go back to the lobby (Or run /trigger bingo.lobby)
Click here to switch to spectator mode (Or run /trigger bingo.spectator)
```

Do you think that is appropriate?

Reply:
Yep